src/marqo/s2_inference/clip_utils.py

```
@@ -424,21 +424,54 @@ def load(self) -> None:
+                    os.remove(self.model_path)
+                except Exception as remove_e:
+                    raise InvalidModelPropertiesError(
+                        f"Marqo encountered an error while attempting to delete corrupted file `{self.model_path}`. "
+                        f"Please manually check and remove the file. Error message: `{str(remove_e)}`"
+                    )
+                raise InvalidModelPropertiesError(
+                    f"Marqo encountered a corrupted file when loading open_clip file `{self.model_path}`. "
+                    f"Marqo has removed this file from the disk. \n"
+                    f"This is likely to be caused by 3 reasons: 1. the file is not a valid open_clip checkpoint, "
+                    f"2. the file is corrupted during download or incomplete download, "
+                    f"3. you may load a `clip` model with `open_clip` model properties. \n"
+                    f"Please check and update your model properties and retry. "
+                    f"You can find more details at `https://docs.marqo.ai/0.0.21/Models-Reference/bring_your_own_model/#bring-your-own-clip-model`")
+
+            elif "This could be because the operator doesn't exist for this backend" in str(e):
+                raise InvalidModelPropertiesError(
+                    f"Marqo encountered an error when loading custom open_clip model `{self.model_name}` with "
+                    f"model properties = `{self.model_properties}`. \n"
+                    f"The error message is `{str(e)}`. \n"
+                    f"It is likely that you are tyring to load a `CLIP (OpenAI)` model with type `open_clip` in model_properties. \n"
+                    f"Please check and update your model properties and retry. "
+                    f"You can find more details at `https://docs.marqo.ai/0.0.21/Models-Reference/bring_your_own_model/#bring-your-own-clip-model`"
+                )
 
-        return model, preprocess
+            else:
```

This else catches all unknown error types, right? 

Then we need to raise an error that returns a 5xx to the user, because we have encountered an unexpected situation, that could be a bug in Marqo.

---

The 5xxs also highlight bugs for us to further investigate